### PR TITLE
Compatibility with Rails 4.2

### PR DESCRIPTION
**Pull request by bquorning**

### CODE REVIEW

File: lib/arturo/special_handling.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@ def blacklist(feature_symbol = nil, &block)
       def two_arg_block(symbol, block)
         return block if symbol.nil?
         lambda do |feature, recipient|
-          feature.symbol == symbol && block.call(recipient)
+          feature.symbol.to_s == symbol.to_s && block.call(recipient)
```
Comment (bquorning) [inline lib/arturo/special_handling.rb]: Without this change, Rails 4.2 will compare strings with symbols here.

File: test/dummy_app/test/unit/features_helper_test.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
 require File.expand_path('../../test_helper', __FILE__)
 require 'arturo/features_helper'
+require 'rails-dom-testing'
 
 class ArturoFeaturesHelperTest < ActiveSupport::TestCase
 
   include ActionView::Helpers::TagHelper
-  include ActionDispatch::Assertions::SelectorAssertions
+  include Rails::Dom::Testing::Assertions::SelectorAssertions
```
Comment (bquorning) [inline test/dummy_app/test/unit/features_helper_test.rb]: `ActionDispatch::Assertions::SelectorAssertions` is extracted into the `rails-dom-testing` gem in Rails 4.2.

**Comment by bquorning** [comment]
cc @grosser , @jamesarosen

**Comment by jamesarosen** [comment]
+1

**Comment by grosser** [comment]
:+1:

**Comment by bquorning** [comment]
Does that mean you will merge it? :-)

**Comment by jamesarosen** [comment]
Two +1s certainly does!

**Pull request merged**